tests/Klasse.spec.ts

```diff
@@ -156,4 +187,146 @@ test.describe(`Testfälle für die Administration von Klassen: Umgebung: ${proce
       await klasseManagementView.checkTableData();
     });
   });
+  test('Klasse bearbeiten als Landesadmin', { tag: [LONG, SHORT, STAGE] }, async ({ page }) => {
+    const header = new HeaderPage(page);
+    const landing: LandingPage = new LandingPage(page);
+    const login: LoginPage = new LoginPage(page);
+
+    let userInfoAdmin: UserInfo;
+    const startseite = new StartPage(page);
+    const menue = new MenuPage(page);
+    const klasseManagementView = new KlasseManagementViewPage(page);
+    const klasseCreationView = new KlasseCreationViewPage(page);
+    const klasseDetailsView = new KlasseDetailsViewPage(page);
```

Comment:
Bitte dies hier löschen und im nächsten Step mit FromAnywhere(page).start() anfangen und von dort aus die jeweiligen Methoden nutzen, um die gewünschte Seite klasseCreationView zu returnen. Beispiel in Import.spec.ts ab Z31.

menue kann bestehen bleiben, damit navigiert werden kann